build.sbt

```diff
@@ -1,8 +1,9 @@
 name := "codacy-detekt"
 
 scalaVersion := "2.13.1"
+kotlinVersion := "1.5.21"
```

Comment:
Is this some leftover ?

Reply:
It is to force the kotlinVersion on plugin

Reply:
So you are setting the `kotlinVersion` which is already defined in the Kotlin SBT Plugin.
Got it. It seemed to me you had defined an unused variable.